packages/code-assist/README.md

```diff
@@ -34,7 +34,7 @@ Google Maps Platform resources that the MCP server can access include:
 
```

Comment:
Instead of "Developers" as the heading, how about "Why use this?" or "Benefits" which seems to describe the content within better.